src/clients/index.js

```diff
@@ -0,0 +1,4 @@
+import ws from '@cesarbr/knot-cloud-websocket';
```

Comment:
Regarding the object name, I have two points:
1. Use the name in PascalCase, as it refers to a class
2. I knot that we have been calling it simply "websocket", but what it actually is, is the web socket interface to the protocol adapter. As now we are going to have multiple services in the same library (and we could have web socket interfaces to other services), I think it would be less confusing if we name it by the service it interacts with (we can keep the WS as part of the name. Not sure if `ProtocolAdapterWebSocket` is the best name though. Do you have any suggestions?

Reply:
It could be this the name. We were considering changing the library name to `knot-cloud-sdk-protocol-adapter-websocket` or something like that, right?

Reply:
Done. I'm following what we've discussed before.